internal/errcheck.go

```diff
@@ -63,6 +64,21 @@ func (e byName) Less(i, j int) bool {
 	return ei.line < ej.line
 }
 
+// findPackage is similar to the default implementation (loader.defaultFindPackage),
+// but allows local imports like "./foo" to resolve relative to the current directory.
+func findPackage(ctxt *build.Context, path string) (*build.Package, error) {
+	wd, err := os.Getwd()
```

Comment:
You're right that cwd cannot change, during the process (it wouldn't make sense to use it here if it did), but I doubt that it is worth factoring it out - the cost is negligible in comparison to everything else that must be done for each package.